commands/command_prune.go

```diff
@@ -288,7 +284,10 @@ func pruneTaskCollectErrors(outtaskErrors *[]error, errorChan chan error, errorw
 	}
 }
 
-func pruneDeleteFiles(prunableObjects []string, task *tasklog.PercentageTask) {
+func pruneDeleteFiles(prunableObjects []string, logger *tasklog.Logger) {
+	task := tasklog.NewPercentageTask("prune: Deleting objects", uint64(len(prunableObjects)))
+	logger.Enqueue(task)
```

Comment:
Small idea for anther PR:

```go
task := logger.NewPercentageTask(...) // create and enqueue task
defer task.Complete()
```

Reply:
🤦‍♂️  I'm silly -- this exists! Changed in 65b4f34e.